### DISCUSSION TITLE
【求助】救一下，急急急

### DISCUSSION THREAD

**Discussion by AnkaAlioth** [share121/inter-knot#2796] [其他]
![image](https://github.com/user-attachments/assets/d4de7b57-2da6-443e-9aef-fed72f74cdc0)

各位绳网用户大家好，我是法厄同的小号。
如隔壁帖所说我被治安局抓了#2776，现在急需**2000菲林**办理保释

急需好心人救一下，我妹妹已经在零号空洞等着了
等我这次摆脱了治安局，就把Fairy的端口公开出来给大家用。
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/651fba00ba55004182fffbb5aa553c53_6086077287436441482.png" alt="米仔_膜拜" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/651fba00ba55004182fffbb5aa553c53_6086077287436441482.png" alt="米仔_膜拜" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/651fba00ba55004182fffbb5aa553c53_6086077287436441482.png" alt="米仔_膜拜" width="75" >

**Comment by SKAIZH** [comment]
好说好说，友情价一千六百菲林吧

**Reply by AnkaAlioth** [reply]
你不会是刚刚拷我的女警官吧！！

**Comment by Aha-SWYH** [comment]
法厄同在这抛头露面不怕被抓吗

**Reply by AnkaAlioth** [reply]
你好，已经被抓了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by WXZ1025** [comment]
这个委托我接了🤓👆

**Comment by SKAIZH** [comment]
竟然那么多人吗，我很好奇绳网在线人数有多少

**Comment by KEARUM** [comment]
三百菲林，不能再多了

**Comment by Lanxey** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/10/10/fae60c50fb0bbb804afa4bc67d51e4f7_898755171324052040.png" alt="拉尼_蹲笑" width="75" >你是法厄同那我是谁
